plugins/report-portal/src/components/LaunchesPage/LaunchesPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import {
+  Breadcrumbs,
+  Content,
+  Header,
+  HeaderIconLinkRow,
+  Link,
+  Page,
+  useQueryParamState,
+} from '@backstage/core-components';
+import { useRouteRef } from '@backstage/core-plugin-api';
+
+import AssessmentIcon from '@material-ui/icons/Assessment';
+
+import { projectsRouteRef, rootRouteRef } from '../../routes';
+import { LaunchesPageContent } from './LaunchesPageContent/LaunchesPageContent';
+
+export const LaunchesPage = () => {
+  const rootR = useRouteRef(rootRouteRef);
```

Comment:
```suggestion
  const rootR = useRouteRef(rootRouteRef);
```

use complete readable variables

Reply:
changed the var. names here